<launch>

  <node type="teleop_gripper" pkg="pr2_teleop" name="teleop_gripper_right" output="screen">
    <param name="open_button" type="int" value="13" />
    <param name="close_button" type="int" value="15" />
    <remap from="command" to="r_gripper_controller/command" />
  </node>

  <node type="teleop_gripper" pkg="pr2_teleop" name="teleop_gripper_left" output="screen">
    <param name="open_button" type="int" value="5" />
    <param name="close_button" type="int" value="7" />
    <remap from="command" to="l_gripper_controller/command" />
  </node>

  <node pkg="pr2_teleop" type="teleop_pr2" name="pr2_teleop" output="screen">
    <remap from="cmd_vel" to="base_controller/command" />

    <!-- Axes -->
    <param name="axis_vx" value="3" type="int"/>
    <param name="axis_vy" value="2" type="int"/>
    <param name="axis_vw" value="0" type="int"/>
    <param name="axis_pan" value="0" type="int"/>
    <param name="axis_tilt" value="3" type="int"/>

    <!-- Base velocities -->
    <param name="max_vw" value="0.8" />
    <param name="max_vx" value="0.5" />
    <param name="max_vy" value="0.5" />
    <param name="max_vw_run" value="1.4" />
    <param name="max_vx_run" value="1.0" />
    <param name="max_vy_run" value="1.0" />
 
    <!-- Head -->
    <param name="max_pan" value="2.7" />
    <param name="max_tilt" value="1.4" />
    <param name="min_tilt" value="-0.4" />
    <param name="tilt_step" value="0.3" />
    <param name="pan_step" value="0.3" />

    <!-- Buttons have changed for PS3 controller mapping -->
    <!-- With joystick remapper, this works with all joysticks on "/joy" -->
    <param name="run_button" value="11" type="int" />
    <param name="torso_dn_button" value="14" type="int" />
    <param name="torso_up_button" value="12" type="int" />
    <param name="head_button" value="8" type="int" />
    <param name="deadman_button" value="10" type="int"/>

  </node>

</launch>

